docs/en/Configuration-Settings.md

```diff
@@ -13,15 +13,39 @@ for Alluxio in different contexts.
 
 # Configuration in Alluxio
 
-Alluxio runtime respects two sources of configuration settings:
+Alluxio runtime respects three sources of configuration settings:
 
-1. [Environment variables](#Environment-variables). This is a convenient way for beginner users or shell scripts to set
-a few basic properties when running Alluxio.
-2. [Configuration properties](#configuration-properties). This provides a general way to customize any
-[supported Alluxio configure properties](#appendix).
+1. [Application settings](#application-settings). Setting Alluxio configuration in this way is application-specific,
+and is required each time when running a new Spark/MapReduce job or an Alluxio shell command.
+2. [Environment variables](#environment-variables). This is an easy and fast way to set the basic properties
+to manage Alluxio servers and run Alluxio shell commandlines.
+Note that, configuration set through environment variables may not be realized by applications.
+3. [Property files](#property-files). This is a general approach to customize any
+[supported Alluxio configure properties](#appendix). Configuration in those files can be respected by Alluxio servers,
+as well as applications.
 
-The priority to load a property values, from the highest to the lowest, is
-environment variables, properties files and the defaults.
+
+The priority to load property values, from the highest to the lowest, is
+application settings (if any), environment variables, property files and the defaults.
+
+## Application settings
+
+Alluxio shell users can use `-Dkey=property` to specify an Alluxio configuration value in commandline. For example,
+
+```java
+$ bin/alluxio fs -Dalluxio.user.file.writetype.default=MUST_CACHE touch /foo
```

Comment:
FYI, it is odd that this key contains "default" as a suffix ... we are using it here to override the default